extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -241,13 +242,29 @@ export function buildDom(document, element) {
   measurer.classList.add(MEASURER_CLASS);
 
   realChildNodes(element).forEach((node) => contentWrapper.appendChild(node));
-  measurer./*OK*/ innerHTML = contentWrapper./*OK*/ innerHTML;
+  mirrorNode(contentWrapper, measurer);
   element.appendChild(content);
   element.appendChild(measurer);
 
   return {content, contentWrapper, measurer};
 }
 
+/**
+ * Make a destination node a clone of the source.
+ *
+ * @param {!Node} source
+ * @param {!Node} dest
+ */
+function mirrorNode(source, dest) {
+  // First clear out the destination node.
+  removeChildren(dest);
+
+  const {childNodes} = source;
```

Comment:
nit: add a comment for:
`// Then copy all the source's child nodes  into destination node.`